packages/apex-node/src/tests/asyncTests.ts

```diff
@@ -310,12 +311,10 @@ export class AsyncTests {
     }
 
     const queryPromises = queries.map(query => {
-      return this.connection.tooling.autoFetchQuery(query) as Promise<
-        ApexTestResult
-      >;
+      return this.connection.tooling.query<ApexTestResult>(query);
     });
     const apexTestResults = await Promise.all(queryPromises);
-    return apexTestResults;
+    return (apexTestResults as unknown) as ApexTestResult[];
```

Comment:
Three query changes in the test....all slightly different 🥇

Reply:
We talked about getting rid of the `as unknown`